lib/improver/tests/blending/blend_across_adjacent_points/test_TriangularWeightedBlendAcrossAdjacentPoints.py

```diff
@@ -34,22 +34,37 @@
 import unittest
 
 from cf_units import Unit
```

Comment:
Several unused imports at top of this file (manually doing codacy checks). They haven't been added by you but we could tidy them up as we are tidying these files:
from cf_units import Unit
from iris.coords import DimCoord
from iris.cube import Cube
from iris.exceptions import CoordinateNotFoundError